src/app/components/Navbar/Navbar.tsx

```diff
@@ -7,17 +7,19 @@ type Props = {
 
 export default function Navbar({ children }: Props) {
   return (
-    <div className="px-4 py-2 bg-white flex justify-between items-center border-b border-gray-200 dark:bg-surface-01dp dark:border-white/10">
-      <div className="flex w-8/12 md:w-4/12 lg:w-3/12">
-        <AccountMenu />
-      </div>
-      {children && (
-        <div>
-          <nav className="flex space-x-8">{children}</nav>
+    <div className="px-4 py-2 bg-white border-b border-gray-200 dark:bg-surface-01dp dark:border-white/10">
+      <div className="max-w-screen-lg flex justify-between mx-auto w-full px-3 items-center">
+        <div className="flex w-8/12 md:w-4/12 lg:w-3/12">
+          <AccountMenu />
+        </div>
+        {children && (
+          <div>
+            <nav className="flex space-x-8">{children}</nav>
+          </div>
+        )}
+        <div className="md:w-4/12 lg:w-3/12 flex justify-end items-center">
+          <UserMenu />
         </div>
-      )}
-      <div className="md:w-4/12 lg:w-3/12 flex justify-end items-center">
-        <UserMenu />
       </div>
     </div>
```

Comment:
recommendation for reviewers: enable the **"hide whitespace"** option above and the diff becomes a LOT easier to read. I added an inner wrapper `<div>` and moved three classes from the existing outermost wrapper to this new inner wrapper (`.flex`, `.justify-between` and `.items-center`).

Reply:
or install:
https://addons.mozilla.org/en-US/firefox/addon/github-whitespace-disabler/
https://chrome.google.com/webstore/detail/github-diff-whitespace-di/bppopebgnjajocjnjlginjfpjbleappc